src/Pages/FieldInfoPage.js

```diff
@@ -1,4 +1,5 @@
 import {
+  Button,
```

Comment:
remove this button

Reply:
"Fetch" button not needed anymore, if its being used in code, remove it. location is filtered based on selected in drop down

Reply:
Done